result-page/result-page.js

```diff
@@ -0,0 +1,58 @@
+import { renderLine } from './render-line.js';
+const resultTable = document.getElementById('result-data');
+const stringyResults = localStorage.getItem('RESULTS');
+const parsedResults = JSON.parse(stringyResults);
+
+parsedResults.forEach((element) => {
+    const newLine = renderLine(element);
+    resultTable.append(newLine);
+});
+
+const labels = [];
+const votes = [];
+
+parsedResults.forEach(item => {
+    labels.push(item.name);
+    votes.push(item.numSelected);
+});
+
+var ctx = document.getElementById('result-graph').getContext('2d');
+/* eslint-disable */
+var myChart = new Chart(ctx, {
+/* eslint-enable */
+    type: 'bar',
+    data: {
+        labels: labels,
+        datasets: [{
+            label: '# of Votes',
+            data: 
+            votes,
+            backgroundColor: [
```

Comment:
I think you only have background color on your first five bars because you haven't designate more here. Same with border color below.